src/main/java/camp/nextstep/edu/calculator/StringCalculator.java

```diff
@@ -0,0 +1,61 @@
+package camp.nextstep.edu.calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.springframework.util.StringUtils;
+
+public class StringCalculator {
+
+    public int calculate(String source) {
+        StringFormula formula = StringFormula.parse(source);
+
+        String[] operands = formula.extractOperands();
+
+        return sum(operands);
+    }
+
+    private int sum(String[] operands) {
+        return Arrays.stream(operands)
+            .mapToInt(PositiveNumberUtils::parsePositiveInt)
+            .sum();
+    }
+
+    static class StringFormula {
+
+        private static final String DELIMITER_REGEX = "[,:]";
+        private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.*)\n(.*)");
+        static final StringFormula EMPTY = new StringFormula(DELIMITER_REGEX, "");
+        static final String[] EMPTY_OPERANDS = new String[0];
+
+        private String delimiter;
+        private String formula;
+
+        private StringFormula(String delimiter, String formula) {
+            this.delimiter = delimiter;
+            this.formula = formula;
+        }
+
+        static StringFormula parse(String source) {
+            if(StringUtils.isEmpty(source)) {
```

Comment:
```suggestion
            if(isEmpty(source)) {
```
아래 메서드가 재사용하지 않고 있는데요. 이를 제거하고 static import 하는 것에 대해서는 어떻게 생각하시나요?